### PR TITLE
Add creation time and task name to submission admin page

### PR DESCRIPTION
**Pull request by danlamanna**

### CODE REVIEW

File: core/admin.py [hunk 1]
```diff
@@ -66,6 +89,9 @@ class SubmissionAdmin(admin.ModelAdmin):
 
     actions = [rescore_submission_with_notification, rescore_submission_without_notifying]
 
+    def task(self, obj: Submission):
+        return obj.approach.task
```
Comment (brianhelba) [inline core/admin.py]: Will this be expensive? Can we prefetch related?

Comment (danlamanna) [reply core/admin.py]: It somehow already is, I'm guessing because approach is in the fields list.

File: core/admin.py [hunk 2]
```diff
@@ -53,11 +57,30 @@ class SubmissionInline(ReadonlyTabularInline):
...
+        if self.value():
+            return queryset.filter(approach__task=self.value())
+        else:
+            return None
+
+
 @admin.register(Submission)
 class SubmissionAdmin(admin.ModelAdmin):
-    list_display = ['approach', 'id', 'status']
+    list_display = ['approach', 'task', 'id', 'created', 'status']
```
Comment (brianhelba) [inline core/admin.py]: I think it'd be more friendly to put the 'task' before the 'approach'.

Comment (brianhelba) [reply core/admin.py]: I just did it.

**Pull request merged**